package.json

```diff
@@ -217,6 +221,7 @@
     "mmjstool": "mmjstool",
     "storybook": "start-storybook -p 6006 -s ./storybook/static",
     "build-storybook": "build-storybook -s ./storybook/static",
-    "check-types": "tsc"
+    "check-types": "npx patch-package && tsc",
```

Comment:
I've added the `patch-package` on the `check-types` command because it is failing on Circle CI's `type-check` stage, maybe because the `node_modules` on `install` stage is not persisted/cached?

Reply:
Strange. I'll ask the person in charge of CircleCI about that. If we can't fix it in CircleCI, then I'm fine leaving this here.

Reply:
Looks like it's just a problem with `npm` not calling `postinstall` sometimes. We have had similar issues elsewhere, so I don't think we need to change anything here